lock-impl/src/main/java/com/palantir/lock/impl/LegacyTimelockService.java

```diff
@@ -82,39 +95,13 @@ public TimestampRange getFreshTimestamps(int numTimestampsRequested) {
 
     @Override
     public LockImmutableTimestampResponse lockImmutableTimestamp() {
-        long immutableLockTs = timestampService.getFreshTimestamp();
```

Comment:
toLockTs? this is no longer just immutableTs

Reply:
I'll FLUP with cleanups in a separatte PR.